### PR TITLE
Add highlights to the 6.10 release notes index page

### PR DESCRIPTION
**Pull request by backmari**
### Description of work
Add highlights to the 6.10 release notes index page. Minor edits.

---

### Reviewer

Please comment on the points listed below ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)).
**Your comments will be used as part of the gatekeeper process, so please comment clearly on what you have checked during your review.** If changes are made to the PR during the review process then your final comment will be the most important for gatekeepers. In this comment you should make it clear why any earlier review is still valid, or confirm that all requested changes have been addressed.

#### Code Review

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?
- Do the release notes conform to the [release notes guide](https://developer.mantidproject.org/Standards/ReleaseNotesGuide.html)?

#### Functional Tests

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

### Gatekeeper

If you need to request changes to a PR then please add a comment and set the review status to "Request changes". This will stop the PR from showing up in the list for other gatekeepers.

### CODE REVIEW

File: docs/source/release/v6.10.0/index.rst [hunk 1]
```diff
@@ -16,7 +16,35 @@ Mantid 6.10.0 Release Notes
 
 We are proud to announce version 6.10.0 of Mantid.
 
-**TODO: Add paragraph summarizing big changes**
+In this release, we changed the way files are loaded into Mantid:
+On IDAaaS, loading files will look into the instrument data cache path ``./data/instrument``
```
Comment (jhaigh0) [inline docs/source/release/v6.10.0/index.rst:20]:
```suggestion
On IDAaaS, loading files will look into the instrument data cache path ``/data/instrument``
```

File: docs/source/release/v6.10.0/index.rst [hunk 2]
```diff
@@ -16,7 +16,35 @@ Mantid 6.10.0 Release Notes
...
+  ``LoadEventNexus`` is generally longer, but workflows using
+  ``CompressEvents`` generally run faster).
+- New option to find peaks using the ratio of variance/mean in
+  :ref:`algm-FindSXPeaksConvolve` - this is a peak finding criterion used in
+  DIALS software (Winter, G., et al. Acta Crystallographica Section D,
+  Structural Biology 74.2 (2018): 85-97).
+- New detector grouping options on the :ref:`interface-indirect-diffraction` interface
+  (including ``All``, ``Groups``, ``Custom`` and ``File`` options)
+- New way to easily load multiple files into :ref:`Elwin Tab <elwin>` of
+  :ref:`Data Manipulation Interface <interface-inelastic-data-manipulation>`
```
Comment (jhaigh0) [inline docs/source/release/v6.10.0/index.rst:47]:
```suggestion
- New detector grouping options on the :ref:`interface-indirect-diffraction` interface
  (including ``All``, ``Groups``, ``Custom`` and ``File`` options).
- New way to easily load multiple files into :ref:`Elwin Tab <elwin>` of
  :ref:`Data Manipulation Interface <interface-inelastic-data-manipulation>`.
```

**Review by jhaigh0 — CHANGES_REQUESTED** [review]
A nice summary well done :) A couple of tiny things I spotted (most importantly getting the data cache path correct).

You'll also need to remove line 15 of the index
```
.. warning:: This release is still under construction. The changes can be found in the nightly builds on the `download page`_.
```

**Comment by sf1919** [comment]
I'm wondering if we need to make it clear what IDAaaS is. Can we use `ISIS Data Analysis as a Service Platform (IDAaaS)` the first time it is mentioned and then use the acronymn `IDAaaS` afterwards?

**Comment by backmari** [comment]
@jhaigh0 @sf1919 Thank you for the comments and having a look at the PR!

**Review by jhaigh0 — APPROVED** [review]

**Pull request merged**